src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -280,6 +290,24 @@ public static class CLIOptions
             JsonKey = "dashboard-version"
         };
 
+        public static readonly CLIOption<string> DashboardFallbackVersionOption = new CLIOption<string>
+        {
+            ArgumentName = "--dashboard-fallback-version",
+            ArgumentShortName = "-fallback-version <version>",
+            ArgumentDescription = $"Project version used as a fallback when no report could be found based on Git information for the Compare feature in reporters. Can be semver, git commit hash, branch name or anything else to indicate what version of your software you're testing.",
```

Comment:
This could use an example. Ex: when the current branch is a new feature branch there does not yet exist a report for this branch. If the current branch is based on the master branch, set 'master' as the fallback version.